packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -812,7 +823,12 @@ const CFDPasswordModal = ({
                         values={{
                             // TODO: remove below condition once deriv x changes are completed
                             type: platform === 'dxtrade' && type_label === 'Derived' ? 'Synthetic' : type_label,
-                            platform: platform === CFD_PLATFORMS.MT5 ? mt5_platform_label : 'Deriv X',
+                            platform:
+                                platform === CFD_PLATFORMS.MT5
```

Comment:
please avoid multi optional check and move it outside this scope and define a switch-case fort it